src/i18n/i18n.service.ts

```diff
@@ -109,7 +112,13 @@ export class Overridable {
 		this.isOverridden = true;
 		this._value = value;
 		if (isObservable(value)) {
-			value.subscribe(v => {
+			// To ensure that there are not multiple subscriptions created for the same observable, we
+			// unsubscribe if a subscription already exists for an observable before creating a new one.
+			if (this.subscription) {
```

Comment:
FYI @Donisius gotta put this back outside the `if (isObservable(value)) {`, sorry!